### PR TITLE
DB Tests refactoring

### PR DESCRIPTION
**Pull request by Nabila63Ahmed**

### CODE REVIEW

File: signup-in-out.iml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (HatemMorgan) [inline signup-in-out.iml]: Please add **.iml** file to **.gitignore**

File: src/main/java/database/ArangoHandler.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package database;
+
+
+import java.util.List;
+
+public class ArangoHandler implements DatabaseHandler{
```
Comment (HatemMorgan) [inline src/main/java/database/ArangoHandler.java]: what is the responsibility of this class ?

File: src/main/java/database/DatabaseInstance.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
+package database;
+
+/**
+ * A singleton class carrying a database instance
+ */
+public class DatabaseInstance {
```
Comment (HatemMorgan) [inline src/main/java/database/DatabaseInstance.java]: What does this class do ?

File: database/tableCreation.sql [hunk 4]
```diff
@@ -9,5 +9,4 @@ CREATE TABLE IF NOT EXISTS users(
   id int  AUTO_INCREMENT PRIMARY KEY,
```
[the remark this answers is not in the document]
Comment (ahmedouda1995) [reply database/tableCreation.sql]: removed it.

File: database/userInsertProcedure.sql [hunk 5]
```diff
@@ -3,42 +3,52 @@ USE linkedin;
```
[the remark this answers is not in the document]
Comment (ahmedouda1995) [reply database/userInsertProcedure.sql]: removed this too

File: signup-in-out.iml (hunk 1 above)
Comment (ahmedouda1995) [reply signup-in-out.iml]: done

File: src/main/java/database/DatabaseInstance.java (hunk 3 above)
Comment (ahmedouda1995) [reply src/main/java/database/DatabaseInstance.java]: removed was an old file

**Pull request closed without merging**